test/model-inheritance.test.js

```diff
@@ -174,23 +174,25 @@ describe('Model class inheritance', function() {
       customer.should.have.property('customerId', 'c01');
       customer.should.have.property('bio', undefined);
 
-        // The properties are defined at prototype level
+      // The properties are defined at prototype level
       assert.equal(Object.keys(customer).filter(function(k) {
-          // Remove internal properties
+        // Remove internal properties
         return k.indexOf('__') === -1;
       }).length, 0);
       var count = 0;
       for (var p in customer) {
         if (p.indexOf('__') === 0) {
           continue;
         }
-        if (typeof customer[p] !== 'function') {
+        // Please note there is an injected id from User prototype which is a function
+        // On node >= 7, this property does not show up
+        if (typeof customer[p] !== 'function' && p !== 'id') {
```

Comment:
With the model inheritance PR checked in, wouldn't `customerId` override `id` as a primary key? I would expect the behavior in node7 is correct, since `id` shouldn't show up in `ChildModel`